source/shared/core_conn.cpp

```diff
@@ -569,8 +569,8 @@ bool core_is_authentication_option_valid(const char* value, size_t value_len)
 
 namespace {
 
-connection_option const* get_connection_option( sqlsrv_conn* conn, SQLULEN key, 
-                                                     const connection_option conn_opts[] TSRMLS_DC )
+connection_option const* get_connection_option( sqlsrv_conn* conn, _In_ SQLULEN key, 
+                                                     _In_ const connection_option conn_opts[] TSRMLS_DC )
 {
```

Comment:
not needed, but may be needed in the future. Parameters without annotation should be not currently used in the function, but I didn't delete the parameter.